content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -80,7 +80,7 @@ The generated ConfigMap is:
 apiVersion: v1
 data:
   application.properties: |
-    FOO=Bar
+    FOO: Bar
```

Comment:
The intent here is to store data that can become a [`.properties` file](https://en.wikipedia.org/wiki/.properties); these use `=` not `:`.

/hold
We should not make this change @ChetHosey 

Maybe to make this more clear, we could link to https://en.wikipedia.org/wiki/.properties on line 55 so that people can read about the format.

Reply:
I support linking to Wikipedia to avoid confusion. @ChetHosey would you be willing to modify your PR to do this instead?

Reply:
The original PR updated the wrong example. Please see https://github.com/kubernetes/website/pull/30784#issuecomment-991274708 for correction and actual kubectl output vs. the documented version.

I've updated the PR accordingly. Thanks!